CONTACT.md

```diff
@@ -0,0 +1,4 @@
+Welcome to the LunaSea community.
+
+Settle differences by offering and accepting constructive critique, within desired privacy.
+Finding yourselves unable, e-mail Avatar Jagandeep Brar (the project maintainer) at hello@comet.tools
```

Comment:
I don't mind removing the code of conduct and replacing it with this, it was just a recommended thing from GitHub that I didn't think too much about.

Small request before merging, can you update the email to `hello@lunasea.app` and add a period at the end?